drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt

```diff
@@ -5518,9 +5594,77 @@ Internet-Draft            TE Common YANG Types            September 2023
            "RFC 4115 Bandwidth Profile";
          reference
            "RFC4115: A Differentiated Service Two-Rate, Three-Color
+
+
+
+Busi, et al.               Expires 20 May 2024                [Page 100]
+
+Internet-Draft            TE Common YANG Types             November 2023
+
+
            Marker with Efficient Handling of in-Profile Traffic";
        }
 
+     // CHANGE NOTE: The identity path-metric-loss below has
+     // been added in this module revision
+     // RFC Editor: remove the note above and this note
+     identity path-metric-loss {
```

Comment:
thinking a bit, why do we want to prepend "path-" to metric type?
Ideally, it is simply metric-type and this can be per link or per path metric type.
There are cases were we can set a bound applicable on per link (e.g. loss, DV, etc.) and other cases where metric is end-to-end.. I'd think where the metric leaf is will tell if per link or per path.

What do you think?